Vostok.Logging.File/FileLog.cs

```diff
@@ -95,14 +102,22 @@ internal FileLog(IMultiFileMuxer muxer, Func<FileLogSettings> settingsProvider)
             eventsLost = new AtomicLong(0);
 
             filePathProvider = new CachingTransform<FileLogSettings, FilePath>(
-                settings => new FilePath(settings.FilePath), preventParallelProcessing: false);
+                settings => new FilePath(settings.FilePath),
+                preventParallelProcessing: false);
+
+            Instances.Add(this);
         }
 
         /// <summary>
         /// The total number of events dropped by all <see cref="FileLog"/> instances in process due to event queue overflows.
         /// </summary>
         public static long TotalEventsLost => DefaultMuxer.EventsLost;
 
+        /// <summary>
+        /// The number of events dropped by this <see cref="FileLog"/> instance due to events queue overflow.
+        /// </summary>
+        public long EventsLost => eventsLost;
```

Comment:
`Interlocked.Read`?

Reply:
`eventsLost` is a property of `AtomicLong` type, which has `Interlocked.Read` underneath.

This change appeared after reformatting this file, sorry :P